app/services/reports/adult_incontinence_report_service.rb

```diff
@@ -127,7 +127,7 @@ def total_people_served_with_loose_supplies_per_month
                         .distributions
                         .for_year(year)
                         .joins(line_items: :item)
-                        .merge(Item.adult_incontinence)
+                        .merge(Item.adult_incontinence.where(kit_id: nil)) # exclude kits
```

Comment:
@dorner If this is supposed to be loos supplies only I feel that kits need to be excluded. If this is indeed the case it also get the test cases within a fraction of the expected `Adults Assisted Per Month` and `Adult incontinence supplies per adult per month` values. I am unsure how precise the test values are and if they can or should be changed.

Reply:
Hmm -- it *shouldn't* make any difference because kits are not classified as adult incontinence - they are classified as kits.   But you seem to have found that it does?